js/src/dropdown.js

```diff
@@ -174,7 +174,9 @@ class Dropdown extends BaseComponent {
         }
       }
 
+      this._menu.classList.add('popper-active')
```

Comment:
Can you please add a comment here why we add `popper-active` and we remove it after creating the popper?

Reply:
Thanks for this, we don't need to remove this class indeed. I've added a comment about why we need to add this class.

Reply:
Lol, ok, nevermind, we do need this to fix the dropleft positioning 😛